### ISSUE TITLE
Make it clear what the lock is for

### ISSUE THREAD

**Issue by gprst** [arduia/ProExpense#27]
I am still wondering what the lock next to the "Save" button is for when you create an entry, maybe a label or a tutorial might help ?

**Comment by arduia** [comment]
@gprst 
Yes, I assume the Lock Button is at right place where next to Save Button. I just have to add  tutorials for new users with popup.

Thanks for your suggestions it is really help for me.

**Comment by gprst** [comment]
So I understand thanks to your answer on #28 that the lock button is for recurring expenses. Is a lock icon the best ? Shouldn't it be a loop icon, or a calendar one, or something ?

**Comment by arduia** [comment]
@gprst Yes, I haven't decide which icon to choose for lock yet. The loop icon might be suitable for this feature. I will try with this.
